mindmeld/cli.py

```diff
@@ -785,6 +792,36 @@ def _get_auto_annotator_config(app_path, overwrite=False, unannotate_all=False):
     return config
 
 
+@shared_cli.command("augment", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path",
+    required=True,
+    help="The application's path.",
+)
+@click.option(
+    "--lang",
+    help="Augmentation language.",
```

Comment:
Does the language code have to follow a certain format, like ISO 639? If so, its good to mention it here.

Reply:
Good point. ISO 639-1. Added that.